scripts/whitelist-fiscal-code-script/src/utils/get-logger.ts

```diff
@@ -0,0 +1,58 @@
+import { createLogger, transports, format } from 'winston';
+import { getArgvParam } from './get-argv-param';
+import fs from 'fs';
+
+const outputDir = getArgvParam('--outputDir') ?? 'logs';
+
+export const logger = createLogger({
+  level: 'info',
+  transports: [
+    new transports.Console({
+      format: format.combine(
+        format.timestamp({
+          format: 'YYYY-MM-DD HH:mm:ss', // Timestamp format
+        }),
+        format.printf(
+          ({ timestamp, level, message, stack }) =>
+            `${timestamp} [${level}]: ${message} ${stack || ''}`,
+        ),
+      ),
+    }),
+    new transports.File({
+      filename: outputDir + '/app.log',
+      level: 'info',
+      format: format.combine(
+        format.timestamp(),
+        format.printf(({ timestamp, level, message }) => {
+          return `${timestamp} [${level}]: ${message}`;
+        }),
+      ),
+    }),
+  ],
+});
+
+fs.truncateSync(outputDir + '/whitelisted-fiscal-codes.csv', 0);
+
+export const whitelistedFiscalCodeFileLogger = createLogger({
+  level: 'info',
+  transports: [
+    new transports.File({
+      filename: outputDir + '/whitelisted-fiscal-codes.csv',
+      level: 'info',
+      format: format.printf(({ message }) => String(message)),
+    }),
+  ],
+});
+
+fs.truncateSync(outputDir + '/not-whitelisted-fiscal-codes.csv', 0);
+
+export const notWhitelistedFiscalCodeFileLogger = createLogger({
+  level: 'info',
+  transports: [
+    new transports.File({
+      filename: outputDir + '/not-whitelisted-fiscal-codes.csv',
+      level: 'info',
+      format: format.printf(({ message }) => String(message)),
+    }),
+  ],
+});
```

Comment:
I think this way is very complicated. There is all the complexity of the logs that is not needed for this task. You can simply use 
```ts
fs.appendFileSync(filePath, rowLine, 'utf8'); 
```

to append to an existing csv file (or create it if it doesn't exist).